### ISSUE TITLE
Add List-Unsubscribe Header to decrease SPAM detection

### ISSUE THREAD

**Issue by denniserdmann** [contao/newsletter-bundle#27]
I'm using the contao newsletter in combination with amazon SES and found out, that some email providers like United Internet (GMX, web.de) block my newsletters, when the List-Unsubscribe Header is missing. Furhermore, some email clients like thunderbird detect the newsletter as spam.

**My quick and dirty solution:**
i added the List-Unsubscribe Header by myself 
```
$objEmail->addHeader('List-Unsubscribe', '<mailto:'.$objNewsletter->sender.'?subject=Newsletter-Abmeldung>');
```

[here](https://github.com/contao/newsletter-bundle/blob/master/src/Resources/contao/classes/Newsletter.php#L331).

As it may be possible to put this into an extension, i think it's better to have this in `contao-newsletter` itself. What do you think?

**Comment by Ernestopheles** [comment]
I added an unsubscribe link at the bottom of the newsletter via the email-template, will this not be sufficient?
I know there are some good reasons to have an unsubscribe-header but this does not convince me that much.

**Comment by denniserdmann** [comment]
> I added an unsubscribe link at the bottom of the newsletter via the email-template, will this not be sufficient?

No, it wasn't (i have one too). I tested my newsletter with [mail-tester.com](https://mail-tester.com) and had a 10 out of 10 rating. There were only 2 things that were not perfect:

1. no DMARK
2. no List-Unsubscribe-Header

As i added the second, all mails got through the spam filter. When i removed it some days later, my newsletters got blocked again.
